instrumentation/opentelemetry-instrumentation-falcon/tests/app.py

```diff
@@ -46,12 +47,17 @@ def on_get(self, _, resp):
 
 
 def make_app():
-    if hasattr(falcon, "App"):
+    _parsed_falcon_version = package_version.parse(falcon.__version__)
+    if _parsed_falcon_version >= package_version.parse("3.0.0"):
         # Falcon 3
         app = falcon.App()
-    else:
+    elif _parsed_falcon_version >= package_version.parse("2.0.0"):
         # Falcon 2
         app = falcon.API()
+    else:
+        # Falcon 1
+        app = falcon.API()
+
```

Comment:
do you need this branch? v1 and v2 have the same `falcon.API()`.

Reply:
Nice call, carry over from earlier code. Changed.